src/directives/sortable.js

```diff
@@ -96,9 +102,13 @@ ngeo.sortableDirective = function($timeout) {
               dragListGroup.dispose();
             }
 
+            if (options['dragHoverClass'] !== undefined) {
+              dragHoverClass = options['dragHoverClass'];
+            }
+
             dragListGroup = new goog.fx.DragListGroup();
             dragListGroup.addDragList(element[0],
-                goog.fx.DragListDirection.DOWN);
+                goog.fx.DragListDirection.DOWN, true, dragHoverClass);
```

Comment:
No I used it in a former solution. The class name passed to through this option is added to the list when the user start to drag and till the end of the drag event. My first solution was to increase the height of the list with an additional CSS class. I though that this parameter might be useful in the future.